UPDATING.md

```diff
@@ -70,6 +70,12 @@ https://developers.google.com/style/inclusive-documentation
 
 -->
 
+### Removed pod_launcher from core airflow
+
+Moved the pod launcher from `airflow.kubernetes.pod_launcher` to `airflow.providers.cncf.kubernetes.utils.pod_launcher`
+
+This will alow users to fix the KubernetesPodOperator without requiring an airflow upgrade
```

Comment:
```suggestion
This will allow users to release new versions of Kubernetes provider without requiring an airflow upgrade
```

Reply:
I fixed this doc with slightly different wording.